### ISSUE TITLE
OpenSource : Docker Hub Repository 

### ISSUE THREAD

**Issue by ashish-patwal** [girlscript/winter-of-contributing#3799]
## Description
Docker Hub is a service provided by Docker for finding and sharing container images with your team. It is the world’s largest repository of container images with an array of content sources including container community developers, open source projects and independent software vendors (ISV) building and distributing their code in containers.

##  Checklist 
- Explain Docker Hub .
- Explain Role of Docker Hub in distributing and shipping opensource software .
- Using Docker Hub with example .
- Installing Docker and using images from docker hub .
- Advantages / Disadvantages of docker hub .
- Alternatives to Docker Hub .

### Domain

Open Source

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by ELLIPSIS009** [comment]
i would like to work on this

**Comment by ELLIPSIS009** [comment]
@ashish-patwal @Aryamanz29 can you manually assign me this issue because the bot wont let me because my other pr are merged but the issue has not yet been closed by the mentors

**Comment by ashish-patwal** [comment]
@ELLIPSIS009 can you do /assign . See if that works .

**Comment by ELLIPSIS009** [comment]
i tried that but the bot is glitched so it doesnt work

**Comment by ashish-patwal** [comment]
@ELLIPSIS009 it says above that you have some open issues already assigned to you . Please resolve those issues before taking up other issues.

**Comment by ELLIPSIS009** [comment]
actually i have done with my part in those issue and my pr is also merged but the issue was not closed by the supervisor/mentor so thats the problem

**Comment by ashish-patwal** [comment]
@ELLIPSIS009 i cannot assign you this issue as i am not a maintainer only a mentor . I will ping @Aryamanz29 .

**Comment by ashish-patwal** [comment]
PTAL @Aryamanz29

**Comment by riyapandey730** [comment]
I want to work on this

**Comment by ashish-patwal** [comment]
Use /assign to assign yourself this issue .

**Comment by ELLIPSIS009** [comment]
@ashish-patwal acceding to FCFS this issue will be assigned to me. am i right @Aryamanz29 ?

**Comment by ashish-patwal** [comment]
@ELLIPSIS009 yes you're right . But we can let others work on this issue since assign bot is not letting us give you this issue and neither the maintainer is active right now . Opensource isn't about competing and we'll be posting many other issues as well . Reminder that this event will last till December so you'll get a whole lot of issues to work upon .

**Comment by ELLIPSIS009** [comment]
actually i have started working on the documentation so that's why

**Comment by Aryamanz29** [comment]
@ELLIPSIS009  Assigned happy docs 🎉🎉, Add your doc to `Misc` folder

**Comment by Aryamanz29** [comment]
@Meghna2080  @riyapandey730  You can grab this one :arrow_right:  https://github.com/girlscript/winter-of-contributing/issues/4038

**Comment by aaditya2801** [comment]
/assign 

can I contribute related to docker hub?
